<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordWidget</class>
 <widget class="QWidget" name="WordWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WordWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="labelWord">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Word:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelDefinition">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Definition:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="textDefinition"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="textWord"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelRelated">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Related:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelExamples">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Examples:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="textRelated"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPlainTextEdit" name="textExamples">
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>labelWord</zorder>
  <zorder>labelRelated</zorder>
  <zorder>textExamples</zorder>
  <zorder>labelDefinition</zorder>
  <zorder>textRelated</zorder>
  <zorder>textDefinition</zorder>
  <zorder>labelExamples</zorder>
  <zorder>textWord</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>textWord</tabstop>
  <tabstop>textDefinition</tabstop>
  <tabstop>textRelated</tabstop>
  <tabstop>textExamples</tabstop>
 </tabstops>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections/>
</ui>
